### PR TITLE
feat(gov): update genesis with new voting params

### PR DESCRIPTION
**Pull request by tbruyelle**

### CODE REVIEW

**Review by giunatale — CHANGES_REQUESTED** [review]
Can you also update the Readme to reflect the change?

**Comment by tbruyelle** [comment]
> Can you also update the Readme to reflect the change?

done!

File: README.md [hunk 1]
```diff
@@ -23,7 +23,10 @@ This base genesis includes the following changes:
 - **x/mint**: inflation disabled, no new $GOVGEN token minting
 - **x/gov**: 
 	- deposit amount raised to 5000 $GOVGEN, 
-	- voting period extended to 365 days
+	- voting period is replaced by 3 new different voting periods:
+        - for text proposal: 365 days
```
Comment (giunatale) [inline README.md]:
```suggestion
           - for text proposal: 365 days
```

File: README.md [hunk 2]
```diff
@@ -23,7 +23,10 @@ This base genesis includes the following changes:
...
+        - for parameter change proposal: 14 days
```
Comment (giunatale) [inline README.md]:
```suggestion
           - for parameter change proposal: 14 days
```

File: README.md [hunk 3]
```diff
@@ -23,7 +23,10 @@ This base genesis includes the following changes:
...
+        - for software upgrade proposal: 28 days
```
Comment (giunatale) [inline README.md]:
```suggestion
           - for software upgrade proposal: 28 days
```

**Review by giunatale — CHANGES_REQUESTED** [review]

**Comment by tbruyelle** [comment]
A new parameter will be added for the default voting period, let's not merge this PR until that.

**Comment by tbruyelle** [comment]
> A new parameter will be added for the default voting period, let's not merge this PR until that.

done here [4c94023](https://github.com/atomone-hub/govgen-genesis/pull/1/commits/4c940239e500fdc1a1a0de987370ce10e56a70c6) & [d462ad7](https://github.com/atomone-hub/govgen-genesis/pull/1/commits/d462ad75ff0f43e75f315acfa9598f52610407e2)

File: base-genesis.json [hunk 4]
```diff
@@ -108,7 +108,10 @@
         "max_deposit_period": "1209600s"
       },
       "voting_params": {
-        "voting_period": "31536000s"
+        "voting_period_default": "172800s",
```
Comment (giunatale) [inline base-genesis.json]: I would make it 14 days minimum tbh.
2 seems low

**Review by giunatale — CHANGES_REQUESTED** [review]

File: base-genesis.json (hunk 4 above)
Comment (tbruyelle) [reply base-genesis.json]: Yeah I agree, I forgot to think about the value, which is the one defined for the default period the legacy module. Will have to update https://github.com/atomone-hub/govgen/pull/16 first.

Comment (tbruyelle) [reply base-genesis.json]: oh silly me, we can just replace it there.

Comment (tbruyelle) [reply base-genesis.json]: done [1140a7b](https://github.com/atomone-hub/govgen-genesis/pull/1/commits/1140a7b633497b2cdee9647f05ecc3a70e271eb7)

File: README.md [hunk 5]
```diff
@@ -23,7 +23,11 @@ This base genesis includes the following changes:
 - **x/mint**: inflation disabled, no new $GOVGEN token minting
 - **x/gov**: 
 	- deposit amount raised to 5000 $GOVGEN, 
-	- voting period extended to 365 days
+	- voting period is replaced by 3 new different voting periods:
+           - for text proposal: 365 days
+           - for parameter change proposal: 14 days
+           - for software upgrade proposal: 28 days
+           - if the proposal is none of the above: 2 days
```
Comment (giunatale) [inline README.md]: need to fix here too and we are good to go

**Review by giunatale — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (tbruyelle) [reply README.md]: done ty!

**Review by giunatale — APPROVED** [review]

**Pull request merged**